Makefile

```diff
@@ -73,7 +73,7 @@ fmt:
 test: test.gno test.go test.flappy
 	@echo "Full test suite finished."
 
-test.gno: test.files1 test.files2 test.realm test.packages test.examples
```

Comment:
why removing the `test.realm` rule from the `test.gno` group?

Reply:
realm tests are included in files1 and files2?

Reply:
how yes, you're right, they are included in `test.files2` :)